restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/asr/asr.adoc

```diff
@@ -0,0 +1,13 @@
+[[intro]]
+= Automatic Speech Recognition Integration
+
+In order to enable ASR functionality an appropriate configuration has to be added. There are list of available drivers supported by mediaserver and a default one, which can be used by Restcomm. These are environment variables you need to set in restcomm.conf file
+
```

Comment:
why don't we put those options in https://github.com/RestComm/Restcomm-Connect/blob/master/restcomm/configuration/config-scripts/as7-config-scripts/restcomm/mediaserver.conf instead of restcomm.conf ?